components/dashboard/src/start/StartWorkspace.tsx

```diff
@@ -77,7 +77,7 @@ export default class StartWorkspace extends React.Component<StartWorkspaceProps,
         properties: {
           sessionId,
           instanceId: this.state.workspaceInstance?.id,
-          workspaceId: this.state?.workspace?.id,
+          workspaceId: this.state?.workspaceInstance?.workspaceId,
```

Comment:
@akosyakov  `workspaceInstance` is pulled async, that's why it might be undefined.
To avoid `undefined` as much as possible use: `this.props.workspaceId` (always set) and `this.state?.startedInstanceId` (set [immediately after start](https://github.com/gitpod-io/gitpod/blob/edefe8782fbf2281ee2930da90d2349c2c1fde79/components/dashboard/src/start/StartWorkspace.tsx#L112-L122))

Reply:
thanks! it fixed now